hack/windows/Prepare-AntreaAgent.ps1

```diff
@@ -32,8 +32,12 @@ if ($AntreaHnsNetwork) {
     }
 }
 if ($NeedCleanNetwork) {
+    $ovsRunMode = "service"
+    if ($RunOVSServices -eq $false) {
+        $ovsRunMode = "container"
+    }
     Write-Host "Cleaning stale Antrea network resources if they exist..."
-    & $CleanAntreaNetworkScript
+    & $CleanAntreaNetworkScript -OVSRunMode $ovsRunMode
```

Comment:
We called `$CleanAntreaNetworkScript` is just to clean-up antrea-hns network. If we add "remove OVS service" here, it may break the prepare-AntreaAgent script in this scenario:  for the first time, we call "./Prepare-AntreaAgent.ps1 -RunOVSServices $false",  for the second time , we call "./Prepare-AntreaAgent.ps1 -RunOVSServices $true". The second run would fail because OVS services are removed.

If the purpose is just to remove OVS services (e.g. we want to update Windows host from running OVS as service to running inside container), we can use a separate call "./Clean-AntreaNetwork.ps1 -RemoveOVS $true -OVSRunMode service"

Reply:
I'll remove the call to `RemoveOVSService` in `Clean-AntreaNetwork.ps1`